e2e/utils/TestUtils.ts

```diff
@@ -23,6 +24,7 @@ export function createDefaultSettings() {
 function createSettingsFile(settingsDir: string) {
```

Comment:
Can we get rid of `createSettingsFile` because it's only called from `createDefaultSettings`, and all `createDefaultSettings` does is call `createSettingsFile`? I'd rather reduce that surface. And maybe change the name so we know `createDefaultSettings` is called only by e2e tests so we don't have to worry that user data is getting overwritten (like by the line turning on kubernetes).

Reply:
- The path is `e2e/utils/…`, I'm pretty certain it's e2e-only
- It _does_ overwrite my settings, because we don't override the global config.